spec/faraday/log_filter/filter_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "spec_helper"
+
+describe Faraday::LogFilter::Filter do
+  describe "#filter_url" do
+    subject { described_class.new([:foo, token: :truncate, api_key: :truncate]) }
+
+    let(:url) do
+      URI("http://foo.bar?token=sometoken&format=json&api_key=12345677890&foo=bar")
+    end
+
+    it "replaces the filtered param value with [FILTERED]" do
```

Comment:
it looks like this spec is actually testing both filtering and truncation, so perhaps the description could be changed to reflect this, such as:

`filters out one of the parameters and truncates the other`

or something to that effect